modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -11,14 +11,14 @@ Expanding the cluster requires a DHCP server. Each node must have a DHCP reserva
 [IMPORTANT]
 .Reserving IP addresses so they become static IP addresses
 ====
-Some administrators prefer to use static IP addresses so that each node's IP address remains constant in the absence of a DHCP server. To configure static IP addresses with NMState, see "(Optional) Configuring host network interfaces in the `install-config.yaml` file" in the "Setting up the environment for an OpenShift installation" section for additional details.
+Some administrators prefer to use static IP addresses so that each node's IP address remains constant in the absence of a DHCP server. To configure static IP addresses with NMState, see "Optional: Configuring host network interfaces in the `install-config.yaml` file" in the "Setting up the environment for an OpenShift installation" section for additional details.
 ====
 
 Preparing the bare metal node requires executing the following procedure from the provisioner node.
 
 .Procedure
 
-. Get the `oc` binary, if needed. It should already exist on the provisioner node.
+. Optional: Get the `oc` binary:
```

Comment:
If the oc binary should already exist, why would it be optional to get it? 
If we want to have the line in here, maybe we should just tell the user to confirm it's there?